docs/product-migration-guide/src/main/asciidoc/migration-non-execution-server-con.adoc

```diff
@@ -0,0 +1,19 @@
+[id='migration-non-execution-server-con']
+ifdef::DM[]
+= Migrating {CENTRAL} data
+
+You must migrate your {CENTRAL} data from {PRODUCT_OLD} 6.4 to {PRODUCT} {PRODUCT_VERSION} using the {CENTRAL} migration tool provided with this release to accommodate a new data structure in {PRODUCT} {PRODUCT_VERSION}. You must also update the {PRODUCT_OLD} 6.4 database to be used with {PRODUCT} {PRODUCT_VERSION} using one of the upgrade scripts provided with this release to accommodate a database schema change {PRODUCT} {PRODUCT_VERSION}.
+endif::DM[]
+
+ifdef::PAM[]
+= Migrating {CENTRAL} as design environment only
+
+If you use {CENTRAL} to design decision services but do not use {CENTRAL} as an execution server with live tasks and process instances in {PRODUCT_OLD} 6.4., then follow the steps in this section to migrate to {PRODUCT} {PRODUCT_VERSION}.
```

Comment:
6.4, (remove .)

Reply:
No, it's what I mean. {PRODUCT_OLD} 6.4 = BPMS 6.4, which is intended.